app/styles/_shame.scss

```diff
@@ -13,7 +13,17 @@
 .data-table-pagination-left {
   @extend .au-c-help-text;
 }
-
+.helptext {
```

Comment:
I would name the classes consistent with the other ones, so `help-text`, `link-external` and `active-status`